app/containers/Header/styledComponents/listItem.js

```diff
@@ -0,0 +1,8 @@
+import styled from 'styled-components';
```

Comment:
Do not create subfolder for styled components
file name needs to start from capital letter